confluent_kafka/schema_registry/error.py

```diff
@@ -17,21 +17,25 @@
 
 from fastavro.schema import SchemaParseException, UnknownType
 
-__all__ = ['SchemaRegistryClientError', 'SchemaParseException', 'UnknownType']
+__all__ = ['SchemaRegistryError', 'SchemaParseException', 'UnknownType']
 
 
-class SchemaRegistryClientError(BaseException):
+class SchemaRegistryError(BaseException):
     """
-        Error thrown by Schema Registry clients
+    Error thrown by Schema Registry clients
```

Comment:
i would say it more "represents and error returned by Confluent Schema Registry". if you need another exception type for errors generated by the client (i suspect not), then have another type for that.

Reply:
or use out-of-the-box exception types